### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by lwindolf**
Hi!

I like the plugins people write for Gedit, like yours!

And I also am very frustrated with 3rd party plugin finding and installation in Gedit and wrote a plugin to simplify it. It is not perfect as it is AFAIK not possible to safely activate the plugin after installing, but I hope this can be solved in the future.

For now please consider this PR as a suggestion to propagate an easier 3rd party plugin installation.

Note that gedit-plugininstaller does several things:

- download + installation from github
- checking for pip3
- check + installation of Python modules
- enabling the plugin
- reminding the user about restarting gedit

I'd be happy about any feedback!

With Best Regards,
Lars

### CODE REVIEW

**Review by IpshitaC — COMMENTED** [review]
Hi!
I wonder if you could consider opening an issue for this [here](https://github.com/coala/coala-gedit/issues) and then linking this PR to the issue? 
Also, feel free to ping the community on [Gitter](https://gitter.im/coala/coala) and we'll be happy to discuss this 😄

**Comment by jayvdb** [comment]
blocked on moban

**Comment by lwindolf** [comment]
@IpshitaC Sorry that it took me so long. As suggested I added a issue for this PR

File: README.md [hunk 1]
```diff
@@ -1,5 +1,11 @@
 Installation
 ------------
 
-Just run `./install.sh` to copy the files into the right places. Feel free to
-submit a Pull Request featuring a cleaner way!
+For easy automatic installation you can install this plugin using the [Gedit Plugin Installer](https://github.com/lwindolf/gedit-plugininstaller/) or you can do it manually:
```
Comment (jayvdb) [inline README.md]: " or you can do it manually:" .. "For manual installation run" is a bit repetitive.

IMO just remove " or you can do it manually:"

Comment (jayvdb) [reply README.md]: Also break the lines to be <= 80cpl.

File: README.md [hunk 2]
```diff
@@ -1,5 +1,11 @@
...
-Just run `./install.sh` to copy the files into the right places. Feel free to
-submit a Pull Request featuring a cleaner way!
+For easy automatic installation you can install this plugin using the [Gedit Plugin Installer](https://github.com/lwindolf/gedit-plugininstaller/) or you can do it manually: 
+
+For manual installation run 
+
+    pip install
+    ./install.sh
+    
+to copy the files into the right places. Feel free to submit a Pull Request featuring a cleaner way!
```
Comment (jayvdb) [inline README.md]: put the second sentence on a new line.

**Review by jayvdb — CHANGES_REQUESTED** [review]
@lwindolf , would you be so kind to update your commit message to follow http://coala.io/commit

```
README.md: Mention gedit-plugininstaller

Closes https://github.com/coala/coala-gedit/issues/26
```

File: README.md (hunk 1 above)
Comment (lwindolf) [reply README.md]: Done. Although 80cpl is not fully possibly with Markdown link.

File: README.md (hunk 2 above)
Comment (lwindolf) [reply README.md]: Done.

**Comment by lwindolf** [comment]
Commit message also adapted.

**Pull request closed without merging**